articles/azure-app-configuration/rest-api.md

```diff
@@ -5,14 +5,14 @@ author: maud-lv
 ms.author: malev
 ms.service: azure-app-configuration
 ms.topic: reference
-ms.date: 08/17/2020
+ms.date: 11/28/2022
 ---
 
 # Azure App Configuration REST API
 
-High-level documentation on the REST API for Azure App Configuration is available in the main [Azure REST documentation](/rest/api/appconfiguration/). This section contains more details for developers working with the App Configuration REST API.
+The documentation on the [control plane](/azure/azure-resource-manager/management/control-plane-and-data-plane#control-plane) REST API for Azure App Configuration is available in the [Azure REST documentation](/rest/api/appconfiguration/). This section contains details for developers working with the [data plane](/azure/azure-resource-manager/management/control-plane-and-data-plane#data-plane) REST API of Azure App Configuration.
 
-The following reference pages are available to describe the Azure App Configuration API surface in detail.
+The following reference pages describe the Azure App Configuration data plane API surface. Unless specified otherwise, all API endpoints assume the endpoint of your App Configuration store, for example, `https://{your-store-name}.azconfig.io`.
```

Comment:
> Unless specified otherwise, all API endpoints assume the endpoint of your App Configuration store, for example, `https://{your-store-name}.azconfig.io`.

This sounds like a definition of the data plane API. Perhaps it should be moved up above where data plane is used for the first time. A callout that the data plane is the API is the one made available at the endpoint of the configuration store.

Reply:
Updated. Please let me know if it looks better now.